DoctorWho.DB/Models/Episode.cs

```diff
@@ -16,7 +16,7 @@ public class Episode
 
         //FK for Doctor 
         public int DoctorId { get; set; }
-
+        public int AuthorId { get; set; }
```

Comment:
before , I Made relation between Episode and Author without having authorId in EAPISODE CLASS ( AUTHOR -- EPISODE  is 1-m relation )
in the requirement AuthorId is required to be entered ,
so I needed to add it here , so mapping will work. then I had another problem in episode seed , so I added AuthorId there .